joomla/joomla-cms
# [5.2] [feature] Media manager select last uploaded item (iframe)

Pull Request for Issue # .

### Summary of Changes

- Select the last uploaded file when media manager is inside an iframe (ie, when selecting an image in a media field or inside an editor)

### Testing Instructions
- Create a new article
- Select an intro image
- navigate to any folder
- Upload an image

The image should be selected

Repeat the same with the CMS Content->Media button in tinyMCE editor

Repeat the same with the media button bellow the Codemirror


### Actual result BEFORE applying this Pull Request

The file was not selected after upload, user needed to pick the file manually

### Expected result AFTER applying this Pull Request

The file is selected after upload, user needed to pick the file manually

### Link to documentations
Please select:
